code/modules/mob/living/carbon/human/species/working_joe/_emote.dm

```diff
@@ -6,3 +6,5 @@
 	var/category = ""
 	/// Override text for the emote to be displayed in the WJ emote panel
 	var/override_say = ""
+	/// Is used by Hazard Joes.
+	var/hazard = FALSE
```

Comment:
this should just be an optional voice line which is used if the emote is being used by a hazard joe, see the original feedback. no need to have two sets of emotes (and therefore keybindings) given they're the same lines

https://github.com/cmss13-devs/cmss13/pull/3919#pullrequestreview-1534751384